examples/triangle/main.rs

```diff
@@ -12,16 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+extern crate cgmath;
 #[macro_use]
 extern crate gfx;
-extern crate gfx_window_glutin;
-extern crate glutin;
+extern crate gfx_app;
```

Comment:
I'm sorry, but we'd prefer to keep `triangle` example to be straight without basing on `gfx_app`.